File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/set_server.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class SetServer extends BridgeCommand {
  final String _ipTerminal;
  final int _portaTransacao;
  final int _portaStatus;

  SetServer(this._ipTerminal, this._portaTransacao, this._portaStatus)
      : super('SetServer');

  @override
  functionParameters() {
    return '"ipTerminal"'
        ':'
        '"$_ipTerminal"'
        ','
        '"portaTransacao"'
        ':'
        '$_portaTransacao'
        ','
        '"portaStatus"'
        ':'
        '$_portaStatus';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/Commands/SetServer.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class SetServer(val ipTerminal: String, val portaTransacao: Int, val portaStatus: Int) :
    BridgeCommand("SetServer") {
    override fun functionParameters(): String {
        return "\"ipTerminal\"" + ":" + "\"" + ipTerminal + "\"" + "," +
                "\"portaTransacao\"" + ":" + portaTransacao + "," +
                "\"portaStatus\"" + ":" + portaStatus
    }
}